app/controllers/concerns/exception_handler.rb

```diff
@@ -0,0 +1,13 @@
+module ExceptionHandler
+  extend ActiveSupport::Concern
+
+  included do
+    rescue_from ActiveRecord::RecordNotFound do |e|
+      json_response({ message: e.message }, :not_found)
```

Comment:
Since modules are not loosely coupled, it is not recommended to do such a construction.